columnq/src/encoding/mod.rs

```diff
@@ -26,6 +28,7 @@ impl TryFrom<&[u8]> for ContentType {
             b"application/arrow.stream" | b"application/vnd.apache.arrow.stream" => {
                 Ok(ContentType::ArrowStream)
             }
+            b"application/parquet" => Ok(ContentType::Parquet),
```

Comment:
given that parquet is also an apache project, we could follow the same pattern that arrow adopts, i.e. support both `application/parquet` and `application/vnd.apache.parquet`

Reply:
Yes, I agree